subsys/bluetooth/host/id.c

```diff
@@ -1284,7 +1280,7 @@ int bt_id_delete(uint8_t id)
 	if (IS_ENABLED(CONFIG_BT_CONN)) {
 		int err;
 
-		err = bt_unpair(id, NULL);
+		err = bt_unpair(id, BT_ADDR_LE_ANY);
```

Comment:
Why is this changed?

Reply:
Using NULL caused memory issues in some circumstances

Reply:
Fixed the issue now